src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1053,7 +1048,8 @@ static void visitHelper (
     std::function<void (std::shared_ptr<SLE> const&)>& function,
```

Comment:
I know it's slightly out of the way, but a `std::function` called `function` bothers me in the same way as a `std::vector` called `vector`.  Why not call it `callback` or `continuation` because that's what it is?